src/main/java/com/iridium/iridiumskyblock/database/IslandPermission.java

```diff
@@ -16,7 +16,7 @@
     @DatabaseField(columnName = "id", generatedId = true, canBeNull = false, uniqueCombo = true)
     private int id;
 
-    @DatabaseField(columnName = "permission", canBeNull = false)
+    @DatabaseField(columnName = "permission", canBeNull = false, unique = true)
```

Comment:
This should be `uniqueCombo`, right? Multiple islands can have the same permission.